model.py

```diff
@@ -166,6 +168,64 @@ class SoftplusLayer(tf.keras.layers.Layer):
     def call(self, inputs):
         return tf.math.log(1 + tf.exp(inputs))
 
+def cross_val_with_best_model(
+    model_fn: Callable,
+    sp: ScaledPartitions,
+    n_cv_folds: int,
+    epochs_per_fold: int,
+    batch_size: int,
+    callbacks_list: List[Callable]):
+    '''
+        Runs cross validation on the model, returning the history and fitted best model. 
+
+        real: Function that builds, compiles, and returns a model.
+        sp: Stands for "Scaled Partitions", representing your dataset. This function only looks at the "train" 
```

Comment:
Is the intended use-case here that most folks using cross-validation will use a `ScaledPartitions` object with only a training set?